libs/atqr/domain/src/lib/EmailService/email.interface.ts

```diff
@@ -1,34 +1,41 @@
-import { EmailAddress } from './emailAddress';
+import { IsEmail } from 'class-validator';
+import { ValueObject } from '../../utils/valueObject';
+
+export class EmailAddress {
+  @IsEmail()
+  emailAddress: string;
+}
 export interface IEmail {
```

Comment:
We don't need an email interface. Remove it and make sure it is not used anywhere